docs/app/Examples/elements/Icon/IconSet/IconCategoryExample.js

```diff
@@ -3,14 +3,14 @@ import React, { PropTypes } from 'react'
 import { Grid, Icon } from 'stardust'
 
 const columns = (icons) => _.map(icons, icon => (
-  <Grid.Column key={icon}>
+  <Grid.Column key={icon} style={{ textAlign: 'center' }}>
     <Icon name={icon} style={{ fontSize: '2em' }} />
     <div>{_.startCase(icon)}</div>
   </Grid.Column>
 ))
 
 const IconCategoryExample = ({ category }) => (
-  <Grid className='center aligned doubling five column'>
+  <Grid className='left aligned doubling five column'>
```

Comment:
`left aligned` in the default, should be able to remove this provided it is not overriding a parent style.